### PR TITLE
rescript-tea

### PR DESCRIPTION
**Pull request by OceanOak**

### CODE REVIEW

File: .bsb.lock [hunk 1]
```diff
@@ -0,0 +1 @@
+1912
```
Comment (pbiggar) [inline .bsb.lock:1]: I think this file shouldn't be checked in.

File: package.json [hunk 2]
```diff
@@ -1,46 +1,39 @@
...
-    "watch:test:client": "watchify --standalone=Test_client --outfile=lib/js/test-ocaml/app_test_client.js lib/js/test-ocaml/test_client.js",
-    "watch:test": "run-p watch watch:test:*",
-    "prebuild:test": "run-s build",
-    "build:test": "run-p build:test:*",
-    "pretest": "run-s build:test",
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "dual-syntax": "bin/dual-syntax"
-  },
+  "name": "rescript-tea",
+  "version": "0.6.1",
```
Comment (pbiggar) [inline package.json:3]: I'm not sure why this file changes, it should be pretty much the same as it was before (with some adaptations for rescript)

**Review by pbiggar — CHANGES_REQUESTED** [review]
This is great! Can you remove .bsb.lock and fix up package.lock?

File: package.json [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
   "version": "0.15.0",
   "description": "TEA for Bucklescript",
   "main": "index.js",
   "scripts": {
-    "clean": "bsb clean",
-    "clean:all": "bsb -clean-world",
-    "build": "bsb",
-    "build:all": "bsb -make-world",
-    "watch": "bsb -w",
+    "clean": "rescript clean",
```
Comment (pbiggar) [inline package.json:8]: Can you make these build targets work?

We can also remove the pre-commit and dual-syntax stuff, since we don't have that anymore.

Comment (OceanOak) [reply package.json:8]: rescript clean -with-deps should do the same thing as bsb -make-world

**Pull request merged**